tinygrad/runtime/ops_metal.py

```diff
@@ -41,22 +41,21 @@ def unwrap(x):
   return ret
 
 @diskcache
```

Comment:
@wozeparrot probably don't need this for metal_ops? because we're letting Metal.framework take care of the cache.